src/__configuration__/markdown/markdownMapping.tsx

```diff
@@ -88,16 +85,21 @@ const Headline: React.FC<Headline> = ({
                 color={'primary'}
                 component={'span'}
                 {...otherTypographyProps}
-                style={{ hyphens: 'auto', display: 'flex', ...otherTypographyProps.style }}
+                className={'headline-text'}
+                style={{ hyphens: 'auto', ...otherTypographyProps.style }}
             >
                 {otherTypographyProps.children}
-                <LinkIcon
-                    color={'action'}
-                    style={{ marginLeft: 16, alignSelf: 'center' }}
-                    titleAccess={'copy to clipboard'}
-                    fontSize={'inherit'}
-                    {...otherSvgIconProps}
-                />
+                <IconButton
+                    onClick={(): void => {
+                        copyTextToClipboard(`${window.location.origin}${window.location.pathname}#${hash}`);
+                        setOnCopy(true);
+                    }}
+                    style={{ position: 'relative', right: -8 }}
```

Comment:
What's with position relative? And why right: -8 instead of just a left margin?

Reply:
I had some issue with that, but it looks like it fixed itself after I dropped flexbox stuff. 

I changed it.